cli/src/main/scala/scala/scalanative/cli/ScalaNativeCli.scala

```diff
@@ -0,0 +1,42 @@
+package scala.scalanative.cli
+
+import scala.scalanative.build.Build
+import scala.scalanative.util.Scope
+import scala.scalanative.cli.utils.ConfigConverter
+import scala.scalanative.cli.utils.NativeConfigParserImplicits._
+import scala.scalanative.cli.options.CliOptions
+import caseapp.core.app.CaseApp
+import caseapp.core.RemainingArgs
+import scala.scalanative.cli.options.BuildInfo
+
+object ScalaNativeCli extends CaseApp[CliOptions] {
+
+  override def ignoreUnrecognized: Boolean = true
+
+  def run(options: CliOptions, args: RemainingArgs) = {
+    if (options.misc.version) {
+      println(BuildInfo.nativeVersion)
+    } else if (options.config.main == None) {
+      println("Required option not specified: --main")
+      exit(1)
+    } else {
+      val ignoredArgs = args.all.filter(_.startsWith("--"))
```

Comment:
We can iterate over unparsed args only once. We should also use `-` instead of `--` for abbreviated args, eg. `-o` instead of `--output`
```suggestion
      val (ignoredArgs, classpath) = args.all.partition(_.startsWith("-"))
```